### ISSUE TITLE
Typescript error "Exports and export assignments are not permitted in module augmentation" (TS2666)

### ISSUE THREAD

**Issue by AxelTerizaki** [Zazama/node-id3#91]
Hello.

I just updated to node-id3 0.2.0 but I get a TS error when trying to compile my own app : 

```node_modules/node-id3/index.d.ts(356,4): error TS2666: Exports and export assignments are not permitted in module augmentations.```

Additionally I tried to use the promise versions but I'm using imports, and tried ti with

```
import {Promise as id3} from 'node-id3';
```

But I got the following error : 

```
src/utils/id3tag.ts(1,9): error TS2305: Module '"node-id3"' has no exported member 'Promise'.
```

Any ideas?

**Comment by Zazama** [comment]
Hey,
seems like my IDE falsely added this import without me noticing, the error is fixed in 0.2.1.
I've also added the Promise Object into the TypeScript definition file.

**Comment by AxelTerizaki** [comment]
Wonderful, it builds now! 

Thanks for the fix. I can't test it right now but at least it compiles, so I'll consider this particular issue closed. :)
